rdflib/util.py

```diff
@@ -456,27 +466,68 @@ def get_tree(
     return (mapper(root), sorted(tree, key=sortkey))
 
 
+_InputT = TypeVar("_InputT")
+_OutputT = TypeVar("_OutputT")
+
+
+def _convert_optional(
+    converter: Callable[[_InputT], _OutputT],
+    input: Optional[_InputT],
+) -> Optional[_OutputT]:
```

Comment:
Actually I thought a bit about this, it is not worth the cost of a function call for this I think, so I will eliminate it, and it is easy enough to do do with `z = None if x is None else convert(x)`

Reply:
Done now.